CMakeLists.txt

```diff
@@ -72,7 +72,7 @@ cotire(bald_engine)
 enable_testing()
 file(GLOB TEST_SRC_FILES ${PROJECT_SOURCE_DIR}/tests/*.cpp src/Log.cpp)
 
-add_executable(runUnitTests ${TEST_SRC_FILES})
+add_executable(runUnitTests ${TEST_SRC_FILES} tests/test_vector3.cpp)
```

Comment:
Not needed here, I know CLion adds it by itself, but remember about it please 😄

Reply:
fixed